webapp/lib/view.dart

```diff
@@ -917,10 +914,10 @@ class ConversationSummary with LazyListViewItem {
       ..classes.add('conversation-selector')
       ..title = 'Select conversation'
       ..checked = _checked
-      ..style.visibility = 'hidden'
+      ..hidden = _checkboxVisible
```

Comment:
hidden = visible ?
Based on the code below, perhaps this should be `..hidden = false` ?
And maybe the field initialization above should be `bool _checkboxVisible = true;` to match ?

Reply:
Thanks @danrubel, yeah, this was a bit messy. Switched to a `_checkboxHidden` variable which makes things more straighforward.